src/visualizer/algorithm/stepbysteptemplate/bubbleInsertionAndShellSortStepByStep.js

```diff
@@ -1,4 +1,4 @@
-export const bubbleAndInsertionSortStepByStep = (animationArr, idx, referenceArray) => {
+export const bubbleInsertionAndShellSortStepByStep = (animationArr, idx, referenceArray) => {
```

Comment:
I think the naming for this should just be `defaultStepByStep`? If not it may get too long if other sorts are also using this